src/methods/create.ts

```diff
@@ -47,6 +48,7 @@ export async function create<S extends PropertiesSchema>(properties: Configurati
         intersectTokenScores: properties.components?.algorithms?.intersectTokenScores ?? intersectTokenScores,
       },
     },
+    [kInsertions]: 0,
```

Comment:
Since Lyra is fully functional, we lost the `kInsertions` reference.

This would add the symbol in the internal interface so we can track the insertions. That is if it is still necessary to do so, of course.